src/scripts/main.js

```diff
@@ -1 +1,66 @@
 'use strict';
+
+const body = document.querySelector('body');
+
+const promise1 = new Promise((resolve, reject) => {
+  body.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(reject, 3000);
+});
+
+const promise2 = new Promise((resolve) => {
+  body.addEventListener('click', () => {
+    resolve('Second promise was resolved');
+  });
+
+  body.addEventListener('contextmenu', () => {
+    resolve('Second promise was resolved');
+  });
+});
+
+const promise3 = new Promise((resolve) => {
+  let leftClick = '';
+  let rightClick = '';
+
+  body.addEventListener('mousedown', () => {
+    const x = event.buttons;
```

Comment:
The `event` object is used here without being defined or passed to the event handler function. You should pass `event` as a parameter to the event listener function to access it correctly.